### PR TITLE
ConfigManager

### PR DESCRIPTION
**Pull request by Z1Meme**

### CODE REVIEW

**Comment by Z1Meme** [comment]
@QuentinGruber Idk how to fix the "Test Binaries" test, I added js-yaml and that seems to be the cause

**Comment by QuentinGruber** [comment]
> @QuentinGruber Idk how to fix the "Test Binaries" test, I added js-yaml and that seems to be the cause

Sorry didn't saw that, will be back tomorow and fix this if it think of it. It's just a "false" require to add so when the exe is build it include the file

**Comment by Z1Meme** [comment]
@QuentinGruber this is ready to merge when the test binaries test is fixed

File: h1z1-server
Comment (QuentinGruber) [inline h1z1-server:1]: ?

File: package.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
     "h1z1-server-demo-2016": "scripts/h1z1-server-demo-2016.js"
   },
   "dependencies": {
+    "@types/js-yaml": "^4.0.5",
```
Comment (QuentinGruber) [inline package.json:16]:
```suggestion
    "@types/js-yaml": "4.0.5",
```
Use fixed version ( so it need to redo the package-lock )

**Review by QuentinGruber — CHANGES_REQUESTED** [review]

**Comment by QuentinGruber** [comment]
i'm looking at the binaries test

**Review by QuentinGruber — APPROVED** [review]

**Pull request merged**